query/config.go

```diff
@@ -53,6 +54,8 @@ type Config struct {
 	OffsetParam string
 	// SearchOperator used to combine search condition together. defaults to "AND".
 	SearchOperator string
+	// SplitValueOnComma used to split the values on comma in the filter values in order to read multiple columns
+	SplitValueOnComma bool
```

Comment:
please move this field to the [Builder type](https://github.com/Stratoscale/swagger/blob/master/query/builder.go#L56) and make it private field.